### PR TITLE
scss-setup

### PR DESCRIPTION
**Pull request by sweetcuptea**
adding CSS files to the projects and using it globally by configuring it on vite config and main js. 

also, I decide to delete the previous code on #3 about the way am importing the fonts.
because I am using fonts.scss now and the code on #3 is not the correct method to import scss files

### CODE REVIEW

File: src/assets/scss/fonts.scss
Comment (trihargianto) [inline src/assets/scss/fonts.scss:1]: So what is the difference between this font.scss and what you did in #3?

Comment (trihargianto) [reply src/assets/scss/fonts.scss:1]: I thought the fonts can be used directly once you load them on vite.config.js

Comment (sweetcuptea) [reply src/assets/scss/fonts.scss:1]: Sorry, you are right that there is no difference between the code on #3
So I'm going to delete this code on the next commit

`fonts({
      dirs: ["./src/assets/fonts"],
}),`

Comment (trihargianto) [reply src/assets/scss/fonts.scss:1]: Yes please

Comment (trihargianto) [reply src/assets/scss/fonts.scss:1]: Also please update the description. Explain the reason why you decide to change the way you load the fonts

Comment (sweetcuptea) [reply src/assets/scss/fonts.scss:1]: done

**Comment by trihargianto** [comment]
LGTM ✅

**Review by trihargianto — APPROVED** [review]

**Pull request merged**